app/src/main/java/org/oppia/app/topic/ViewPagerAdapter.kt

```diff
@@ -27,22 +27,13 @@ class ViewPagerAdapter(
         return TopicInfoFragment.newInstance(internalProfileId, topicId)
```

Comment:
In this file there will some changes:
1. `args` Bundle is not getting used anymore, you can remove that.
2. Also there will be an underline below `when` which should suggesting, `Lift return out of when`, use this option and see the difference.

Reply:
Done